bugsnag-plugin-android-exitinfo/src/main/java/com/bugsnag/android/TombstoneParser.kt

```diff
@@ -28,11 +40,39 @@ internal class TombstoneParser(
             if (listOpenFds) {
                 extractTombstoneFd(trace.openFdsList, fileDescriptorConsumer)
             }
+
+            if (includeLogcat) {
+                extractTombstoneLogBuffers(trace.logBuffersList, logcatConsumer)
+            }
         } catch (ex: Throwable) {
             logger.w("Tombstone input stream threw an Exception", ex)
         }
     }
 
+    private fun extractTombstoneLogBuffers(
+        logBuffersList: List<TombstoneProtos.LogBuffer>,
+        logcatConsumer: (String) -> Unit
+    ) {
+        val newLogList = StringBuilder()
+        logBuffersList.forEach { logs ->
+            logs.logsList.forEach {
+                val priorityType = when (it.priority) {
+                    VERBOSE -> "VERBOSE"
+                    DEBUG -> "DEBUG"
+                    INFO -> "INFO"
+                    WARN -> "WARN"
+                    ERROR -> "ERROR"
+                    ASSERT -> "ASSERT"
+                    else -> it.priority.toString()
+                }
+                newLogList.append(
+                    "\n${it.timestamp} ${it.tid} ${it.tag} $priorityType ${it.message}"
+                )
```

Comment:
I'm surprised our linting didn't pick this up:

```suggestion
                newLogList.append(it.timestamp).append(' ')
                                   .append(it.tid).append(' ')
                                   .append(it.tag).append(' ')
                                   .append(it.priorityType).append(' ')
                                   .append(it.message)
                                   .append('\n')
```